live-examples/css-examples/css/opacity.css

```diff
@@ -0,0 +1,8 @@
+#example-element-container {
+    border: 2px solid;
+    border-image: linear-gradient(90deg, white, turquoise) 1;
```

Comment:
I found this border confusing. I do see the point of wanting to have some background element, but because the container and the example-element are the same size, it looks like the border is on example-element, then it's confusing that the rest of the element disappears but the border does not.

Having played a bit, I think it's probably best just to omit the border, and have the example-element appear and disappear.